src/Components/Menu.module.css

```diff
@@ -52,47 +52,47 @@
 }
 
 @media all and (min-width: 768px) {
-    .navContainer {
-        padding-left: 6.25rem;
-        padding-right: unset;
-        justify-content: unset;
-    }
-    
-    .navElements {
-        gap: unset;
-        padding-left: unset;
-    }
+  .navContainer {
+    padding-left: 6.25rem;
+    padding-right: unset;
+    justify-content: unset;
+  }
 
-    .BookstoreCMS {
-        width: 15rem;
-        height: 2.313rem;
-        margin: 0.375rem 2.938rem 0.125rem 0;
-        font-size: 1.875rem;
-    }
+  .navElements {
+    gap: unset;
+    padding-left: unset;
+  }
 
-    .Books {
-        width: 3.688rem;
-        height: 1rem;
-        margin: 1.125rem 2.563rem 0.688rem 0;
-        font-size: 0.813rem;
-        letter-spacing: 1.9px;
-    }
+  .BookstoreCMS {
+    width: 15rem;
+    height: 2.313rem;
+    margin: 0.375rem 2.938rem 0.125rem 0;
+    font-size: 1.875rem;
+  }
 
-    .Categories {
-        width: 6.563rem;
-        height: 1rem;
-        margin-top: 1.125rem;
-        margin-bottom: 0.688rem;
-        margin-left: 2.563rem;
-        font-size: 0.813rem;
-        letter-spacing: 1.9px;
-      }
+  .Books {
+    width: 3.688rem;
+    height: 1rem;
+    margin: 1.125rem 2.563rem 0.688rem 0;
+    font-size: 0.813rem;
+    letter-spacing: 1.9px;
+  }
 
-      .Logo {
-        width: 2.813rem;
-        height: 2.813rem;
-        margin: 0 0 0 41.5rem;
-        padding: 0.875rem;
-        border: solid 1px var(--white);
-      }
-}
\ No newline at end of file
+  .Categories {
+    width: 6.563rem;
+    height: 1rem;
+    margin-top: 1.125rem;
+    margin-bottom: 0.688rem;
+    margin-left: 2.563rem;
+    font-size: 0.813rem;
+    letter-spacing: 1.9px;
+  }
+
+  .Logo {
+    width: 2.813rem;
+    height: 2.813rem;
+    margin: 0 0 0 41.5rem;
+    padding: 0.875rem;
+    border: solid 1px blue;
+  }
```

Comment:
- [ ] Kindly add a logo, as per the requirement, all your designs should match the given design.